clink/app/src/loader/clink.bat

```diff
@@ -5,6 +5,19 @@
 set clink_profile_arg=
 set clink_quiet_arg=
 
+set clink_autorun=0
```

Comment:
This can be removed by moving the `clink_autorun` test inline into line 15.
...and I'll test to make sure that my observation isn't accidently relying on delayed expansion (which doesn't happen by default).